src/t/d17.clj

```diff
@@ -0,0 +1,258 @@
+(ns t.d17
+  (:gen-class)
+  (:require
+    [clojure.math :as math]
+    [clojure.string :as cljstr]
+    [clojure.set :as cljset]
+    [clojure.core.match :refer [match]]))
+
+(def sample
+"Register A: 729
+Register B: 0
+Register C: 0
+
+Program: 0,1,5,4,3,0")
+(def sol "4,6,3,5,6,3,5,2,1,0")
+
+(def sample2
+"Register A: 2024
+Register B: 0
+Register C: 0
+
+Program: 0,3,5,4,3,0")
+(def sol2 117440)
+
+(defn parse-input
+  [input]
+  (let [ [_ A B C instr] (re-find #"Register A: (\d+)\nRegister B: (\d+)\nRegister C: (\d+)\n\nProgram: ([\d,?]+)" input)
+        instr (->> (cljstr/split instr #",")
+                   (map parse-long)
+                   vec
```

Comment:
[mapv](https://clojuredocs.org/clojure.core/mapv) exists specifically for this use-case:
```suggestion
                   (mapv parse-long)
```